src/cli/src/export.rs

```diff
@@ -115,6 +120,11 @@ pub struct ExportCommand {
     #[clap(long)]
     s3_bucket: Option<String>,
 
+    // The s3 root path
+    /// if s3 is set, this is required
+    #[clap(long)]
+    s3_root: Option<String>,
```

Comment:
I am not sure why we only support s3. Is Are there any reasons that user can't export data into azure blob storage or gcs?

Reply:
because of this
https://github.com/GreptimeTeam/greptimedb/pull/5585#discussion_r1986041379

we need to support it first

Reply:
cc @fengjiachun

Reply:
Once issue #6105 is resolved, we can upgrade the CLI tool to support GCS and AzBlob.